File: ASPNETCore/HowTo/WebApiWPFSample/WebApiWPFSample/src/Modules/Excel/Dtos/CreateRows.dto.cs
namespace WebApiConsoleSample
{
    public class CreateRowsDto : BaseDto
    {
        /**The excel file name that storage manager can recognize. */
        public string ExcelPath { set; get; }
        
        /**The sheet name. Can be null */
        public string SheetName { set; get; }

        /**The row index. */
        public int RowIndex { set; get; }

        /**The count of rows. */
        public int Count { set; get; }
    }
}

File: ASPNETCore/HowTo/WebApiTypescriptSample/src/Modules/Excel/Dtos/CreateRows.dto.ts
import { BaseDto } from "../../../Base/Base.Dto";

export class CreateRowsDto extends BaseDto {
    
    /**The excel file name that storage manager can recognize. */
    public ExcelPath: string;

    /**The sheet name. Can be null */
    public SheetName: string;

    /**The row index. */
    public RowIndex: number;

    /**The count of rows. */
    public Count: number;
}